src/components/SignIn.jsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
-
 import { auth } from '../firebase/firebase';
```

Comment:
This can go with th eimports from line 8-11, Material imports should be on line 5. 

```
import React / Proptypes / withRouter

import withStyles

import firebase / dialog input / styles / etc...
```